.eslintrc.json

```diff
@@ -61,7 +61,14 @@
     "import/no-dynamic-require": "off",
     "import/no-named-as-default": "off",
     "import/no-unresolved": "error",
-    "eslint-plugin-import/no-cycle": "off"
+    "eslint-plugin-import/no-cycle": "off",
+    "sort-imports": [
+        "error",
+        {
+            "ignoreCase": true,
+            "ignoreDeclarationSort": true
```

Comment:
why is this ignored? 🤔

Reply:
I am fine with `false` value but used `true` to make it consistent with [dashbrd](https://github.com/ArdentHQ/dashbrd/blob/develop/.eslintrc.js#L52C10-L52C22), which you would prefer?

Reply:
hm doesn't seem to do what I expected it to do. Will have a look tomorrow what settings I have locally for import sorting as they don't apply with this rule currently